src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -299,50 +299,70 @@ static String stringOrNull(RequestParameter requestParameter) {
 
   static int getTotalCount(JsonObject reportItem) {
     int count = 0;
-    JsonArray itemPerformances = reportItem.getJsonArray("itemPerformance");
+    JsonArray itemPerformances = reportItem.getJsonArray(altKey(reportItem,
+        "itemPerformance", "Performance"));
```

Comment:
Yikes, what horrible crappy work to have to do.

Reply:
Yes.